sopel/cli/run.py

```diff
@@ -703,15 +487,16 @@ def main(argv=None):
             return ERR_CODE
 
         # Step Three: Handle command
-        action = getattr(opts, 'action', 'legacy')
+        action = getattr(opts, 'action', None)
```

Comment:
```suggestion
        action = getattr(opts, 'action', 'start')
```

There was even a comment in the old version of this file:
``# TODO: `start` should be the default in Sopel 8``

Reply:
I'll get back to that to fix that problem. The extra work required here is worth it.